cmd/proxy/main.go

```diff
@@ -135,7 +135,7 @@ func main() {
 	r.Use(cors.New(config))
 
 	r.NoRoute(func(c *gin.Context) {
-		c.JSON(404, gin.H{"message": "not found"})
+		c.File("ui/index.html")
```

Comment:
I think it's not good to replace 404 JSON response with a HTML response here becuase the frontend uses this server as an REST API server. So I suggest we only expose one `/ui` endpoint to serve static web files.

Reply:
I think a better solution here is checking the "Accept" request header.

* Return a JSON response when "Accept: application/json"
* Return a 302 redirection to `/ui` when "Accept: text/html"

Reply:
In this case the sub-routes, for instance `https://localhost:8889/dashboard/overview`, would return not found and could only be accessed through the in-app navigation. Didn't find a better way to solve this. Ideas?

Reply:
@krrprr Get it. It looks like we don't have other better options now. So just keep it like this.

Ref. https://github.com/gin-gonic/contrib/issues/90